### ISSUE TITLE
how to get VAF 

### ISSUE THREAD

**Issue by sophienguyen01** [genome-in-a-bottle/giab_latest_release#17]
Hi I want to filter low VAF from the benchmark vcf file. This is the link to the vcf https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/NA12878_HG001/latest/GRCh38/

How can I calculate the allele frequency using value from the FORMAT column of the vcf : GT:PS:DP:ADALL:AD:GQ

**Comment by jzook** [comment]
Hi Sophie,

Except for ~1000 mosaic variants, which we are currently characterizing, we
expect all variants with GT=0/1 should be VAF=50% and GT=1/1 should be
VAF=100%, since this is a normal sample

**Comment by sophienguyen01** [comment]
Thanks for fast response,

The purpose is I want to get the low frequency variants, but according to what you said here, this vcf does not contain low frequency variants right?
Are the mosaic variants included in this [vcf](https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/NA12878_HG001/latest/GRCh38/HG001_GRCh38_1_22_v4.2.1_benchmark.vcf.gz)?

**Comment by jzook** [comment]
Yeah, we tried to exclude low frequency variants from the benchmark bed and
vcf. We didn't do this perfectly, so we are currently working on
characterizing mosaic variants in HG002 with VAF>5% that we'd missed, and
we should have a draft list of ~100 of these in the next few weeks. It also
appears that ~1000 of the variants in the HG002 v4.2.1 benchmark may be
mosaic with VAF~40-45%, but we're still evaluating these. We're currently
only doing these analyses for HG002, since we have the most data and
analyses for it.
